src/ifcopenshell-python/ifcopenshell/ids.py

```diff
@@ -97,17 +105,23 @@ class classification(facet):
     message = "a classification reference '%(value)s' from '%(system)s'"
 
     def __call__(self, inst, logger):
+        #TODO Location: 'type'/'instance'/'any'
         refs = []
         for association in inst.HasAssociations:
             if association.is_a("IfcRelAssociatesClassification"):
                 cref = association.RelatingClassification
-                refs.append((cref.ReferencedSource.Name, cref.ItemReference))
+                refs.append((cref.ReferencedSource.Name, cref.Identification))  # before was .ItemReference instead of .Identification
```

Comment:
This still needs to be handled in a schema-agnostic way right? ItemReference for 2x3, Identification for IFC4.

Reply:
Of course. Left comment to myself and forgot to reflect both schemas in code. Just pushed a commit with that change.